en/includes/guides/authentication/mfa/add-x509-login.md

```diff
@@ -170,22 +204,20 @@ that you need to use later on in this guide.
     ssl_protocol = "TLS"
     ```
 
-    !!! note
+    !!! Important 
     
-        1.  The `clientAuth` attribute causes the Tomcat to require the client with providing a certificate that can be configured as follows.
+        1.   To function properly, this connector should come first in the order. Otherwise, when mutual SSL takes place, the already existing connector (9443) will be picked up and the certificate will not be retrieved correctly.
```

Comment:
```suggestion
        1.   To function properly, this connector should come first in the order. Otherwise, when mutual SSL takes place, the already existing connector (9443) will be picked up and the certificate will not be retrieved correctly.
```
Let's validate this step. IMO this is not required